src/index.ts

```diff
@@ -198,7 +198,28 @@ export class Client extends EventEmitter {
     }
 
     private generateRandomConnectionUrl() {
-        const knownServerIds = [1, 6, 5, 7];
-        return `wss://ws${knownServerIds[Math.floor(Math.random() * knownServerIds.length)]}.blitzortung.org:3000/`;
+        const knownServerIds = [1, 6, 7, 8];
+        return `wss://ws${knownServerIds[Math.floor(Math.random() * knownServerIds.length)]}.blitzortung.org`;
+    }
+
+    private decode(b: string): string {
```

Comment:
I'm not exactly comfortable with including measures that were clearly taken to obsfuscate the blitzortung API, if the owner of the website does not wish their website be automatically accessed I'm not really willing a game escalating measures.

Further this code was [taken ad-verbatim](https://www.blitzortung.org/en/JS/lbr.js) from the website w/out a license which makes this code questionable to include best.